orchestration/src/main/java/com/sap/ai/sdk/orchestration/AzureContentFilter.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.ai.sdk.orchestration;
+
+import static com.sap.ai.sdk.orchestration.client.model.AzureThreshold.fromValue;
+
+import com.sap.ai.sdk.orchestration.client.model.AzureContentSafety;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+import lombok.val;
+
+@Data
+@Accessors(fluent = true)
+public final class AzureContentFilter implements ContentFilter {
+  @Nullable private Sensitivity hate;
+  @Nullable private Sensitivity selfHarm;
+  @Nullable private Sensitivity sexual;
+  @Nullable private Sensitivity violence;
+
+  @RequiredArgsConstructor
+  public enum Sensitivity {
+    HIGH(0),
+    MEDIUM(2),
+    LOW(4);
+
+    private final int value;
+  }
+
+  @Nonnull
+  com.sap.ai.sdk.orchestration.client.model.FilterConfig toFilterConfigDTO() {
```

Comment:
I tried to consistently label the DTOs as such in the code, so that it's always obvious when we are (preparing) (de-) serialization

Reply:
I was under the impression that we camel-case abbreviations.

`toFilterConfigDto`

Reply:
true, fixed